docs/content/preview/develop/quality-of-service/transaction-priority.md

```diff
@@ -157,3 +160,81 @@ yugabyte=> select * from account;
   </tr>
 
 </table>
+
+The following is another example which describes the usage of `yb_transaction_priority` based on the priority types. A transaction is a high priority one if it includes `SELECT`, `UPDATE` operations and so on.
+
+* From an active [ysqlsh](../../../admin/ysqlsh/#starting-ysqlsh) shell, create a table as follows:
+
+  ```sql
+  CREATE TABLE test_scan (i int, j int);
+  ```
+
+* Set `yb_transaction_priority_lower_bound` and `yb_transaction_priority_upper_bound` to be the same, which forces
+`yb_transaction_priority` to be equal to those two, as its not possible to have a deterministic `yb_transaction_priority`.
+
+  ```sql
+  set yb_transaction_priority_lower_bound = 0.4;
+  set yb_transaction_priority_upper_bound = 0.4;
+  ```
+
+* In a transaction block, perform an insert and view the transaction priority as follows:
+
+  ```sql
+  BEGIN TRANSACTION;
+  INSERT INTO test_scan (i, j) values (1, 1), (2, 2), (3, 3);
+  show yb_transaction_priority;
+  COMMIT;
+  ```
+
+  ```output
+      yb_transaction_priority
+  -------------------------------------------
+   0.400000000 (Normal priority transaction)
+  (1 row)
+  ```
+
+* Setting the yb_transaction_priority variable results in an error, because its a read-only flag. You can verify it as follows:
+
+  ```sql
+  set yb_transaction_priority = 0.3;
+  ```
+
+  ```output
+  ERROR:  parameter "yb_transaction_priority" cannot be changed
+  ```
+
+* In the next transaction block, perform a `SELECT` which results in a high priority transaction.
+
+  ```sql
+  set yb_transaction_priority_lower_bound = 0.4;
+  set yb_transaction_priority_upper_bound = 0.4;
+  BEGIN TRANSACTION;
+  SELECT i, j FROM test_scan WHERE i = 1 FOR UPDATE;
+  show yb_transaction_priority;
+  COMMIT;
+  ```
+
+  ```output
+      yb_transaction_priority
+  -------------------------------------------
+   0.400000000 (High priority transaction)
+  (1 row)
+  ```
+
+* In the final transaction block, set a higher `yb_transaction_priority_upper_bound` and `yb_transaction_priority_lower_bound` value, and perform the same `SELECT` query as the previous one. This transaction type is of the highest priority.
+
+  ```sql
+  set yb_transaction_priority_upper_bound = 1;
+  set yb_transaction_priority_lower_bound = 1;
+  BEGIN TRANSACTION;
+  SELECT i, j FROM test_scan WHERE i = 1 FOR UPDATE;
+  show yb_transaction_priority;
+  COMMIT;
+  ```
+
+  ```output
+      yb_transaction_priority
+  -------------------------------------------
+  Highest priority transaction
+  (1 row)
+  ```
```

Comment:
Add an example to show the transaction priority outside a transaction block. This should show 0.